src/kinds/parseViewDefinition.ts

```diff
@@ -1,7 +1,17 @@
 import jp from "jsonpath";
-import pgQuery from "pg-query-emscripten";
 import { last } from "ramda";
 
+let pgQueryInstance: any;
+const getPgQuery = async () => {
+  if (!pgQueryInstance) {
+    const { default: pgQueryModule } = await import(
```

Comment:
New emscripten version requires async initialisation. Since project target didn't support top-level await, and because view parsing is not always required, I made both importing and initialisation happen on-demand